py/openage/codegen/__main__.py

```diff
@@ -89,7 +89,9 @@ def main():
                           "this causes codegen to abort with an error message "
                           "if the target cache has changed."))
 
-    ap.add_argument("--verbose", "-v", action='count', default=0)
+    ap.add_argument("--verbose", "-v", action='count', default=0,
+                    help=("print more stuff for debugging,"
+                          "repeat for even more output."))
```

Comment:
i added that to document the repeatability of this option, because that is what i had to figure out from the code because `--verbose=3` didn't work...